stdlib/_typeshed/__init__.pyi

```diff
@@ -236,6 +236,10 @@ class SupportsNoArgReadline(Protocol[_T_co]):
 class SupportsWrite(Protocol[_T_contra]):
     def write(self, __s: _T_contra) -> object: ...
 
+# stable
+class SupportsFlush(Protocol):
+    def flush(self) -> object: ...
```

Comment:
I actually feel like this might be a situation where we could be opinionated and say, "Actually, file-like objects are doing the wrong thing if they return a non-`None` object from their `flush()` method; that's pretty weird". But I don't feel strongly about it!

Reply:
For consistency alone, I would use `object` here. Also, it catches the (admittedly very theoretical) problem, where someone calling `flush()` tries to use the return value.

Reply:
Sure, I'm fine with keeping this as `-> object`

Reply:
(We could also start using `Unused` instead of `object` for protocol return types to signal intent better, but this should be done in a separate PR.)